data-analytics/dataflow-python-examples/dataflow_python_examples/data_generation_for_benchmarking.py

```diff
@@ -85,11 +123,11 @@ def __init__(self, bq_schema_filename=None, input_bq_table=None, p_null=0.1, n_k
                 else:
                     with open(bq_schema_filename, 'r') as json_file:
                         self.schema = json.load(json_file)
-            except ValueError:
-                logging.error("Not a valid json file! \n %s", str(ValueError))
+            except ValueError as e:
+                logging.error("Not a valid json file! \n %s", e)
             except AttributeError:
                 logging.error("Could not find gcs file %s", str(bq_schema_filename))
```

Comment:
%s will do the string conversion for you, so you don't need the str() cast.

Reply:
Removing extra str cast.